mlos_bench/mlos_bench/optimizers/convert_configspace.py

```diff
@@ -106,7 +106,6 @@ def _tunable_to_configspace(
             name=tunable.name,
             bounds=(int(tunable.range[0]), int(tunable.range[1])),
             log=bool(tunable.is_log),
-            q=nullable(int, tunable.quantization),
```

Comment:
Need to add back quantization support, right?

Reply:
yep. created #803 to track it, assigned to myself

Reply:
```suggestion
            # TODO: Restore quantization support (#803).
```

Reply:
Added this.